addons/web_editor/static/src/scss/wysiwyg_snippets.scss

```
@@ -766,6 +766,13 @@ body.editor_enable.editor_has_snippets {
 
         we-title {
             display: block;
+        }
+
+        // TODO in master, remove the text-transform rule entirely and rely on
+        // developers writing their label with the intended "design". The
+        // "where" change here was added as a way to minimize the impact of this
+        // fix in stable. That CSS style was breaking translations.
+        :where(html[lang^="en-"]) & we-title {
```

> The "where" change here was added as a way **to minimize the impact** of this fix in stable.

Is that because it does something special regarding the specificity?
What would be the difference with
```suggestion
        html[lang^="en-"] & we-title {
```
?

---

`:where` has indeed a specificity of 0 whatever its content. So here I just did not change the specificity of the original capitalize rule, even though I technically prefixed it with `html[lang^="en-"]`.

Note that I say it is to minimize the impact in stable but in the end, I think it would be the right way the write it in master too (but in master we will just remove it so I did not think that far). And note that I could have written it that way:
`we-title:where(html[lang^="en-"] *) {`, but this would be less efficient since selectors are supposed to be evaluated from right to left.

---

@rdeodoo In fact investigating this a bit more for the 16.0 adaptation, I discovered that the CSS has a `:lang(` pseudo-selector :open_mouth: So here I could have written:
```css
we-title:where(:lang(en)) {
```
- `:lang(en)` matches all element which are in english (itself or an ancestor has a `en-XXX` lang attribute).
- use `:where` to still have no increasing specificity

I'll adapt in 16.0 as soon as I have a solution for that version.